pkg/controller/vitesscluster/reconcile_orchestrator.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2019 PlanetScale Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vitesscluster
+
+import (
+	"context"
+
+	"planetscale.dev/vitess-operator/pkg/operator/update"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	planetscalev2 "planetscale.dev/vitess-operator/pkg/apis/planetscale/v2"
+	"planetscale.dev/vitess-operator/pkg/operator/conditions"
+	"planetscale.dev/vitess-operator/pkg/operator/lockserver"
+	"planetscale.dev/vitess-operator/pkg/operator/orchestrator"
+	"planetscale.dev/vitess-operator/pkg/operator/reconciler"
+	"planetscale.dev/vitess-operator/pkg/operator/results"
+)
+
+func (r *ReconcileVitessCluster) reconcileOrchestrator(ctx context.Context, vt *planetscalev2.VitessCluster) (reconcile.Result, error) {
+	key := client.ObjectKey{Namespace: vt.Namespace, Name: orchestrator.ServiceName(vt.Name)}
+	labels := map[string]string{
+		planetscalev2.ClusterLabel:   vt.Name,
+		planetscalev2.ComponentLabel: planetscalev2.OrcComponentName,
+	}
+	resultBuilder := results.Builder{}
```

Comment:
nit: we try to make this the first line of any reconcile subroutine.

Reply:
Also fixed vtctld (source of copy-pasta).

Reply:
Thanks.